tests/unit/preprocessor/_time/test_time.py

```diff
@@ -2160,65 +2160,5 @@ def test_resample_fails_scalar(self):
             resample_time(cube, day=16)
 
 
-@pytest.fixture
```

Comment:
I moved these tests here to the new `tests/integration/_time/test_time.py` module since these are not unit tests.